apps/frontend/src/pages/profile/libs/components/user-details/libs/components/profile-form/profile-form.tsx

```diff
@@ -42,6 +43,12 @@ const ProfileForm = ({ onSubmit, user }: Properties): React.JSX.Element => {
 				name="lastName"
 				type="text"
 			/>
+			<Checkbox
+				control={control}
+				errors={errors}
+				label="Profile visible"
```

Comment:
Let's make it clearer that there is an action
```suggestion
				label="Make profile visible"
```